### DISCUSSION TITLE
bootstrap.js have ECCN number ? Also is it EAR99?

### DISCUSSION THREAD

**Discussion by ytbuchi** [twbs/bootstrap#37583] [Q&A] [answered]
Hi.
please tell me.
Does bootstrap.js have an ECCN number ? Also is it EAR99?

best regards

**Comment by mdo** [comment, marked as answer]
No, it does not.

**Reply by ytbuchi** [reply]
dear mdo 
Thank you for your answer.

Doesn't bootstrap.js include crypto functions?

best regards
